iphone/Classes/MediaModule.h

```diff
@@ -9,6 +9,11 @@
 #if defined(USE_TI_MEDIAGETAPPMUSICPLAYER) || defined(USE_TI_MEDIAOPENMUSICLIBRARY) || defined(USE_TI_MEDIAAPPMUSICPLAYER) || defined(USE_TI_MEDIAGETSYSTEMMUSICPLAYER) || defined(USE_TI_MEDIASYSTEMMUSICPLAYER) || defined(USE_TI_MEDIAHASMUSICLIBRARYPERMISSIONS)
 #import <MediaPlayer/MediaPlayer.h>
 #endif
+#if IS_SDK_IOS_14
```

Comment:
combine the nested if/endif?
```suggestion
#if IS_SDK_IOS_14 && defined(USE_TI_MEDIAOPENPHOTOGALLERY)
#import <PhotosUI/PHPicker.h>
#endif
```